roles/kubernetes/control-plane/templates/k8s-certs-renew.sh.j2

```diff
@@ -14,7 +14,7 @@ echo "## Restarting control plane pods managed by kubeadm ##"
 {% endif %}
 
 echo "## Updating /root/.kube/config ##"
-/usr/bin/cp {{ kube_config_dir }}/admin.conf /root/.kube/config
+/bin/cp {{ kube_config_dir }}/admin.conf /root/.kube/config
```

Comment:
On CentOS8, cp command is `/usr/bin/cp`
How about just removing `/usr/bin/`?

Reply:
done